cfgov/jinja2/v1/_includes/molecules/line-chart.html

```diff
@@ -0,0 +1,41 @@
+
+{# ==========================================================================
+
+   Chart
+
+   ==========================================================================
+
+   Description:
+
+   Create a line chart molecule with data.
+   See [GHE]/flapjack/Modules-V1/wiki/Chart
+
+   ========================================================================== #}
+
+<div class="chart_wrapper block">
+    {% if value.title %}
+        <h3>{{ value.title }}</h3>
+    {% endif %}
+
+	<div class="svg_wrapper"
+        id="{{ value.element_id }}"
```

Comment:
Can these references to the fields on the block be passed through [Jinja2 `escape`](http://jinja.pocoo.org/docs/dev/templates/#html-escaping)? Otherwise HTML tags entered into those fields would just show up here.